apps/frontend/src/store/data_filters.ts

```diff
@@ -158,12 +182,24 @@ export class FilteredData extends VuexModule {
     this.SELECT_EVALUATIONS([fileID]);
   }
 
+  @Action
+  public select_exclusive_evaluations(fileIDs: FileID[]): void {
+    this.CLEAR_ALL_EVALUATIONS();
+    this.SELECT_EVALUATIONS(fileIDs);
+  }
```

Comment:
Instead of duplicating this function can you just pass an array with one item for the spots that use `select_exclusive_evaluation`?